rfc6962-bis.xml

```diff
@@ -906,23 +979,20 @@ an accepted root certificate.
               </list>
             </t>
             <t>
-              The <spanx style="verb">hash</spanx> must be calculated as defined in <xref target="tree"/>. The <spanx style="verb">tree_size</spanx> must designate an existing v2 STH. Because of skew, the front-end may not know the requested STH. In that case, it will return the latest STH it knows, along with an inclusion proof to that STH. If the front-end knows the requested STH then only <spanx style="verb">leaf_index</spanx> and <spanx style="verb">audit_path</spanx> are returned.
+              The <spanx style="verb">hash</spanx> must be calculated as defined in <xref target="tree_leaves"/>. The <spanx style="verb">tree_size</spanx> must designate an existing v2 STH. Because of skew, the front-end may not know the requested STH. In that case, it will return the latest STH it knows, along with an inclusion proof to that STH. If the front-end knows the requested STH then only <spanx style="verb">inclusion</spanx> is returned.
             </t>
             <t hangText="Outputs:">
               <list style="hanging">
-                <t hangText="leaf_index:">
-                  The 0-based index of the entry corresponding to the <spanx style="verb">hash</spanx> parameter.
-                </t>
-                <t hangText="audit_path:">
-                  An array of base64 encoded Merkle Tree nodes proving the inclusion of the chosen certificate.
+                <t hangText="inclusion:">
+                  A base64 encoded <spanx style="verb">TransItem</spanx> of type <spanx style="verb">inclusion_proof</spanx> whose <spanx style="verb">audit_path</spanx> array of Merkle Tree nodes proves the inclusion of the chosen certificate in the selected STH.
```

Comment:
If two different objects (the TransItem object and its member NodeHash vector) are both called "inclusion_proof", how will the reader know which of those objects we're talking about?
ISTM that it makes sense to give them different (but related) names, rather than have to qualify which object we're talking about every time we mention one of them!